magnetic/Generic/main.c

```diff
@@ -21,7 +21,7 @@
 *
```

Comment:
This whole file is unused by Glk; it is the source file for the terminal-based interpreter.  It really doesn't need to exist in the Glk port, but if it is here, it should be reverted entirely.  My guess is this is from an older version of Magnetic Scrolls that never got updated, since the changes in variable names seem completely unnecessary.

Anyway, no need to worry about this file, since we're really only interested in maintaining Glk versions.